app/components/PathToDocument.js

```diff
@@ -6,36 +6,30 @@ import styled from "styled-components";
 import type { DocumentPath } from "stores/CollectionsStore";
 import Collection from "models/Collection";
 import Document from "models/Document";
+import Checkbox from "components/Checkbox";
 import CollectionIcon from "components/CollectionIcon";
 import Flex from "components/Flex";
 
 type Props = {
   result: DocumentPath,
   document?: ?Document,
   collection: ?Collection,
-  onSuccess?: () => void,
+  checked?: boolean,
```

Comment:
```suggestion
  selected?: boolean,
```

Suggest selected - checked makes presumptions about the UI display